packages/lpd8editor/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Nils Hilbricht <nils -at- hilbricht -dot- net>
+
+_projectname=lpd8editor
+pkgname="${_projectname,,}"
+pkgver=0.0.15
+pkgrel=1
+pkgdesc="A Linux editor for the Akai LPD8 pad controller."
+arch=(x86_64 aarch64)
+url="https://github.com/charlesfleche/${_projectname}"
+license=('MIT')
+depends=('alsa-lib' 'qt5-base' 'qt5-tools' 'qt5-svg' 'qt5-translations')
+makedepends=('cmake' 'pkgconf')
```

Comment:
@cbix and I settled on the convention to quote shell array members only if necessary, e.g. `depends=(alsa-lib ...)`.